cohesity_management_sdk/controllers/idps_controller.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, config=None, client=None, call_back=None):
         self.config = config
 
     def add_active_idp_principals(self):
-        """Does a POST request to /public/idp/principals.
+        """Does a POST request to /public/idps/principals.
```

Comment:
was the a miss from previous release or the URL changed ?

Reply:
The URL has been modified.